fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -283,6 +283,9 @@ public void execute(TUniqueId queryId) throws Exception {
             }
 
             if (parsedStmt instanceof QueryStmt) {
+                QueryStmt queryStmt = (QueryStmt) this.parsedStmt;
+                // match sql
+                matchSql(queryStmt);
```

Comment:
I think all kind of stmt can be blocked? Not only select stmt?
And it can be block in `handleQuery()` of `ConnectProcessor`, so that we even don't need to parse it.